specs/eip4844/p2p-interface.md

```diff
@@ -226,6 +226,8 @@ The request MUST be encoded as an SSZ-container.
 The response MUST consist of zero or more `response_chunk`.
 Each _successful_ `response_chunk` MUST contain a single `BlobsSidecar` payload.
 
+In cases where a slot contains a `BlobSidecar` that does not contain any blobs, but contain non-zero `beacon_block_root` and `beacon_block_slot`,  no `blobs_sidecar` is returned.
```

Comment:
```suggestion
In cases where a slot contains a `BlobSidecar` that does not contain any blobs, but contain non-zero `beacon_block_root` and `beacon_block_slot`,  no `blobs_sidecar` **may be** returned.
```
I mean it wont really break things if a peer returns "empty blobs" blobs_sidecar